Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml

```diff
@@ -149,7 +149,7 @@
 - type: entity
   id: PoweredlightExterior
   description: "A light fixture. Draws power and produces light when equipped with a light tube."
-  suffix: Blue
+  suffix: Blue, Don't Map on Delta V #Delta V: Don't Map on Delta V
```

Comment:
```suggestion
  suffix: Blue
  noSpawn: true # DeltaV - Don't map these
```

Reply:
oh, good call. Forgot all about that.

Reply:
If you're automatically migrating them anyways why bother even allowing them to be mapped at all